net-p2p/createtorrent/createtorrent-1.1.4-r1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Create BitTorrent files easily"
+HOMEPAGE="http://www.createtorrent.com/"
+SRC_URI="http://www.createtorrent.com/${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE=""
+
+DEPEND="dev-libs/openssl"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}"
+
+src_prepare() {
+	eapply_user
+
+	sed -i "s:[[]ssl[]]:[crypto]:" configure.in || die "sed failed..."
```

Comment:
changing `configure.in` (which should be renamed to `configure.ac`) without running `eautoreconf`? seems suspicious